coderd/prometheusmetrics/aggregator.go

```diff
@@ -24,20 +25,23 @@ const (
 	loggerName = "prometheusmetrics"
 
 	sizeCollectCh = 10
-	sizeUpdateCh  = 1024
+	sizeUpdateCh  = 4096
 
 	defaultMetricsCleanupInterval = 2 * time.Minute
 )
 
+var MetricLabelValueEncoder = strings.NewReplacer("|", "\\|", ",", "\\,", "=", "\\=")
```

Comment:
You also need to escape `\` or things like

name: `foo`
value: `bar\,x\=y`

will screw this up

Reply:
I embraced the replacer and added a test case verifying the tricky character.